tests/cdc_test.cc

```diff
@@ -39,6 +41,19 @@ SEASTAR_THREAD_TEST_CASE(test_with_cdc_parameter) {
         auto test = [&] (const sstring& create_stmt, bool expected_cdc_value) {
             e.execute_cql(create_stmt).get();
             assert_cdc(expected_cdc_value);
+            if (expected_cdc_value) {
```

Comment:
All of these checks are deep implementation tests, and may change in later implementation, while the API does't. So I would have made them a separate test - so that if we later change the implementation we can see the API checks continue to succeed, while this implementation test is the only one which will break.

Reply:
Which checks do you have in mind when you say that they check only the implementation? Existance of Log and Description is part of API. Content of Description is also part of API.

Reply:
Ok, I completely missed that all the details of the CDC table are the actual API of this feature, not just implementation details....